src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -251,18 +254,18 @@ const formConfig = {
             serviceBranch: {
               'ui:title': 'Branch of service',
             },
-            serviceDateRange: dateRangeUI(
+            serviceDateRange: currentOrPastDateRangeUI(
               'Service start date',
               'Service end date',
-              'End of service must be after start of service',
```

Comment:
It looks like we can keep this line in to have our custom message which is more specific to this input than the generic one that says "To date must be after from date"

Reply:
Done